applications/smart-distancing/libs/core.py

```diff
@@ -65,11 +65,11 @@ def __process(self, cv_image):
             y1 = box[2]
             obj["centroid"] = [(x0 + x1) / 2, (y0 + y1) / 2, x1 - x0, y1 - y0]
             obj["bbox"] = [x0, y0, x1, y1]
-            obj["centroidReal"]=[(x0 + x1)*w / 2, (y0 + y1)*h / 2, (x1 - x0)*w, (y1 - y0)*h]
-            obj["bboxReal"]=[x0*w,y0*h,x1*w,y1*h]
- 
+            obj["centroidReal"] = [(x0 + x1) * w / 2, (y0 + y1) * h / 2, (x1 - x0) * w, (y1 - y0) * h]
+            obj["bboxReal"] = [x0 * w, y0 * h, x1 * w, y1 * h]
+
         objects_list, distancings = self.calculate_distancing(tmp_objects_list)
-        return cv_image, objects_list, distancings
+        return cv_image, objects_list, distancings, tmp_objects_list
```

Comment:
why did you return `tmp_objects_list` when all of its information is in `objects_list`?

Reply:
[because of it](https://github.com/neuralet/neuralet/pull/51#discussion_r418330214)